mithril-aggregator/src/multi_signer.rs

```diff
@@ -378,12 +399,21 @@ impl MultiSigner for MultiSignerImpl {
             .as_ref()
             .ok_or_else(ProtocolError::UnavailableBeacon)?
             .epoch
-            .offset_to_signer_retrieval_epoch()?;
+            .offset_to_signer_retrieval_epoch()
+            .with_context(|| "Multi Signer can not offset to signer retrieveal epoch while retrieving signers with stake")
```

Comment:
Maybe give the value of the reference beacon?

Reply:
Done :heavy_check_mark: